pkg/migration-tool/reader/reader.go

```diff
@@ -93,5 +117,133 @@ func (rr *RemoteRead) Run(errChan chan<- error) {
 			rr.sigBlockRead <- blockRef
 			blockRef = nil
 		}
-	}()
+	}
+
+	readPromBlocks := func() {
+		// Comment.
```

Comment:
what is this?

Reply:
Sorry, this is to add comments for what the respect funcs do. But now, since we want different readers, so this should no longer be a requirement.